repos/spack_repo/builtin/packages/charliecloud/package.py

```diff
@@ -170,9 +189,23 @@ def configure_args(self):
             args.append("--disable-html")
 
         if self.spec.satisfies("+squashfuse"):
-            args.append(f"--with-libsquashfuse={self.spec['squashfuse'].prefix}")
+            if self.spec.satisfies("@0.40:"):
+                # Version 0.40 uses a new syntax for squashfuse
+                args.append(f"--with-squashfuse-include={self.spec['squashfuse'].prefix}/include")
+                args.append(f"--with-squashfuse-lib={self.spec['squashfuse'].prefix}/lib")
+            else:
+                args.append(f"--with-libsquashfuse={self.spec['squashfuse'].prefix}")
```

Comment:
Random style suggestion. I would reverse the sense of the conditional here, i.e.:
```suggestion
            if self.spec.satisfies("@:0.40"):
                args.append(f"--with-libsquashfuse={self.spec['squashfuse'].prefix}")                
            else:
                # Version 0.40 uses a new syntax for squashfuse
                args.append(f"--with-squashfuse-include={self.spec['squashfuse'].prefix}/include")
                args.append(f"--with-squashfuse-lib={self.spec['squashfuse'].prefix}/lib")
```

because then the shorter branch is first, bringing the `if` and `else` closer together, the reasoning being that the reader would remember the condition better if they get to `else` sooner.